python/tvm/autotvm/tuner/graph_tuning.py

```diff
@@ -0,0 +1,85 @@
+"""Utilities for tuning a whole graph (a set of tasks)"""
+
+import os
+from . import callback, XGBTuner, GATuner, RandomTuner, GridSearchTuner
+from .. import record, task
+
+def tune_tasks(tasks,
+               measure_option,
+               tuner='xgb',
+               n_trial=500,
+               early_stopping=200,
+               log_filename='tuning.log',
+               use_transfer_learning=True,
```

Comment:
I feel use_transfer should be part of the tuner option. i.e.

tuner = autotvm.tuner.XGBoostTuner(xxx, use_transfer), but we can default it to be true

Reply:
We need to specify a history data file for using transfer learning. (`tuner. load_history("conv.log"')`). This cannot be done by default. Currently using a default log file for all tasks may pollute the log file.      So we only do transfer learning from scratch inside a `tune_tasks` call.